requirements/edx/development.txt

```diff
@@ -97,7 +97,7 @@ django-crum==0.7.3
 django-debug-toolbar==1.8
 django-fernet-fields==0.5
 django-filter==1.0.4
-django-ipware==1.1.0
+django-ipware==1.2.0
```

Comment:
Interesting that this version was released but not included in the changelog; it looks like the version was initially set incorrectly due to a merge conflict, then fixed and immediately re-released as 2.0.0.  So this should be basically the same as 2.0.0, which adds Django 2.0 compatibility but predates the API change in 2.1.0.  I'm fine with leaving this as is, but it might be more accurate to change the constraint to `django-ipware>=1.1.2,<2.1.0         # Get the client's real IP address; API change in 2.1.0` and let this get upgraded to 2.0.2.  Then again, given that 2.1.0 accidentally broke backwards compatibility without a changelog mention, I'm not confident that nothing broke in 2.0.1 or 2.0.2; may be best to stick with it as is for now.

Reply:
@jmbowman I am not able to follow your concern, the latest version did not change the API for the used functions `get_ip` and `get_real_ip` . I mean that if we use the latest version it would not break anything (I guess); but it will be unwise to upgrade and still use the deprecated functions instead of the new `get_client_ip`; this is why we are sticking with version 1.

Am I right or did I missed something? please advice

Reply:
Sorry, I was a little confused by the version mentioned in the linked django-ipware issue conversation; I thought the deprecation didn't happen until 2.1.0.  Anyway, my point was that version 1.2.0 is basically version 2.0.0 released with the wrong version number, so it might be better to either go back to the officially released 1.1.6 or go ahead and upgrade to one of the deliberately released 2.x versions (continuing to use the deprecated method for now).